documentation/API_documentation/QoD_API.md

```diff
@@ -0,0 +1,178 @@
+<br>
+
+#### QoD Query for QoS Session
+
+| **Quering QoS Session information** |
+| --------------------------------------- |
+| **HTTP Request**<br> GET\<base-url>/qod-latency-api/v0/sessions/{sessionId}<br>**Query Parameters**<br> No query parameters are defined.<br>**Path Parameters**<br> sessionId: Session id that was obtained from the Create QoS Session operation.<br>**Request Body Parameters**<br> No request body parameters are defined.<br>**Response**<br><br> **200: Session information returned.**<br>  Response body:<br> **ueId:** The identifier of the user equipment.<br>   **asId:** The identifier of the application server.<br>   **uePorts (optional):** The requested port(s) on the user equipment.<br>   **asPort (optional):** The requested port(s) on the application server.<br>   **qos:** Qualifier of the requested QoS profile.<br>   **notificationUri (optional):** URI of the callback receiver.<br>   **notificationAuthToken (optional):** Authentication token for callback API.<br>   **id:** Session ID in UUID format.<br>   **startedAt:** Timestamp of session start in seconds since Unix epoch.<br>   **expiresAt:** Timestamp of session expiration if the session was not deleted in seconds since Unix epoch.<br><br> **401:** Un-authorized. <br> **403:** Forbidden. <br> **404:** Session not found.<br> **503:** Service temporarily unavailable. |
+<br>
+
+#### QoD Delete QoS Session
+
+| **Deleting QoS session** |
+| ---------------------------- |
+| **HTTP Request**<br>  DELETE\<base-url>/qod-api/v0/sessions/{sessionId}<br>**Query Parameters**<br>  No query parameters are defined.<br>**Path Parameters**<br>  sessionId: Session ID that needs to be terminated.<br>**Request Body Parameters**<br>  No request body parameters are defined.<br><br>**Response**<br> **204:** Session deleted<br> **401:** Un-authorized. <br> **403:** Forbidden.<br> **404:** Session not found.<br> **503:** Service temporarily unavailable.|
+
+### 4.4 Errors
+
+Since CAMARA QoD API is based on REST design principles and blueprints, well defined HTTP status codes and families specified by community are followed [[4]](#4).
+
+Details of HTTP based error/exception codes for the QoD API are described in Section 4.3 of each API REST based method.
+Following table provides an overview of common error names, codes, and messages applicable to QoD API.
+
+| No | Error Name | Error Code | Error Message |
+| --- | ---------- | ---------- | ------------- |
+|1	|400 |	INVALID_INPUT |	"Expected property is missing: ueId.msisdn" |
```

Comment:
```suggestion
|1	|400 |	Bad Request|	"Expected property is missing: ueId.msisdn" |
```

Corrected Status Code name.

Reply:
I guess this errorinfo.code should be a detailed error code, and not the text for the http response code.

Reply:
INVALID_INPUT is within the YAML v0.8.0 defined. If it should be changed than later together with the YAML.